reviewrot-cronjob-template.yaml

```diff
@@ -0,0 +1,102 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: code-review-reminder-cronjob-template
+  annotations:
+    openshift.io/display-name: "Code review reminder cronjob"
+    description: "Description"
+    tags: "reviewrot code-review-reminder cronjob"
+message: |
+  Cronjob for Code review reminder will run by a schedule ${SCHEDULE}
+  with secrets ${SECRET_CONFIG}/${CONFIG} and
+  ${SECRET_CERTIFICATE}/${CERTIFICATE} on the image
+  quay.io/pbortlov/reviewrot:latest.
+parameters:
+- description: Owner login
+  name: OWNER
+  required: true
+- description: CPU limit
+  name: CPU_LIMIT
+  required: true
+  value: 200m
+- description: Memory limit
+  name: MEMORY_LIMIT
+  required: true
+  value: 256Mi
+- description: Name of configuration's secret
+  name: SECRET_CONFIG
+  required: true
+- description: Name of reviewrot config file
+  name: CONFIG
+  required: true
+- description: Name of certificate's secret
+  name: SECRET_CERTIFICATE
+  required: true
+- description: Name of certificate file
+  name: CERTIFICATE
+  required: true
+- description: Cron Job * * * * *
+  name: SCHEDULE
+  required: true
+
+objects:
+- kind: CronJob
+  apiVersion: batch/v1beta1
+  metadata:
+    name: ${OWNER}-code-review-reminder-cronjob
```

Comment:
Why does it have to be named after the owner? I see you only use the `OWNER` parameter in the names. Could you make the whole name a parameter instead (and get rid of `OWNER` param)?

Reply:
The parameter `OWNER` is used because in one project can be multiple cronjobs and it is an easy way how to distinguish cronjobs. I can rename it to `NAME` to do it more general.